src/librustc_resolve/lib.rs

```diff
@@ -3361,9 +3361,14 @@ impl<'a> Resolver<'a> {
             _ => "enum",
         };
 
-        let (participle, noun) = match old_binding.is_import() {
-            true => ("imported", "import"),
-            false => ("defined", "definition"),
+        let old_noun = match old_binding.is_import() {
+            true => "import",
+            false => "definition",
+        };
+
+        let new_participle = match binding.is_import() {
```

Comment:
I should use this more consistently. The original (old) definition/import and the new one may have different participles/nouns.